core/point/point.go

```diff
@@ -21,3 +25,34 @@ func (p *Point) X() int64 { return p.x }
 
 // Y returns the y-value.
 func (p *Point) Y() int64 { return p.y }
+
+// Kilo Foxtrot addition below
+// Three point translation methods, one is an alternate
+
+// 1. '*Point' immutable int point, to SGF(String of two letters) Point
+func translateToSGFPt01(pt *Point) string {
+	sgfPtX := string(rune((pt.X()) + 97))
+	sgfPtY := string(rune((pt.Y()) + 97))
+	// two lower case letters
+	// return sgfPtX + sgfPtY
+	return strings.ToLower(sgfPtX) + strings.ToLower(sgfPtY)
+}
+
+// Alternate, in case the incoming point is not immutable
+// 2. 'Point' mutable int point, to SGF(String of two letters) Point
+func translateToSGFPt02(pt Point) string {
+	sgfPtX := string(rune((pt.X()) + 97))
+	sgfPtY := string(rune((pt.Y()) + 97))
+	// two lower case letters
+	// return sgfPtX + sgfPtY
+	return strings.ToLower(sgfPtX) + strings.ToLower(sgfPtY)
+}
+
+// 3. 'p' SGF(two letter string) Point, to '*Point' immutable Int Point
+func translateToINTPt01(sgfPt string) *Point {
```

Comment:
Again, you can make clearer with naming:

func NewFromSGF(pt string) *Point {

}